### PR TITLE
[ZPI-53] - Add study visits to grpc

### PR DESCRIPTION
**Pull request by Remmo1**

### CODE REVIEW

**Review by Pandorianim — APPROVED** [review]
Looks good, but why is question made of answers, shouldn't question also have some kind of description?

File: backend/clap/proto/data_store.proto [hunk 1]
```diff
@@ -151,6 +151,23 @@ message SubmitRatingDraft {
...
 }
 
+message Answer {
+    string description = 1;
+    repeated string files = 2;
+}
+
+message Question {
+    repeated Answer answers = 1;
+}
```
Comment (gcfbn) [inline backend/clap/proto/data_store.proto:161]: Where do we store the actual text of the question?

File: backend/clap/proto/data_store.proto [hunk 2]
```diff
@@ -151,6 +151,23 @@ message SubmitRatingDraft {
...
+
+message StudyVisit {
+    repeated Question questions = 1;
+}
+
+message StudyVisitResponse {
+    repeated StudyVisit visits = 1;
+}
```
Comment (gcfbn) [inline backend/clap/proto/data_store.proto:169]: There can be only one visit (list of question/answers) associated with a specific submission, so I think `repeated` is not needed here.

**Review by gcfbn — APPROVED** [review]
LGTM now

**Pull request merged**